test/jdk/java/util/Formatter/Basic-X.java.template

```diff
@@ -1697,6 +1600,16 @@ public class Basic$Type$ extends Basic {
 
         // Get the supported ids for GMT-08:00 (Pacific Standard Time)
         String[] ids = TimeZone.getAvailableIDs(-8 * 60 * 60 * 1000);
+        // With tzdata2020b, the test fails for the mentioned zones expecting
+        // "PST" but JDK has Zone name "MST" for JRE locale provider.
+        // Therefore excluding them as the test is only looking for a GMT-08:00
+        // time zone ID. See JDK-8254865.
+        final List<String> list =  new ArrayList<String>();
+        Collections.addAll(list, ids);
```

Comment:
I think
```java
final List<String> list =  new ArrayList<>(Arrays.asList(ids));
```
is going to be faster than
```java
final List<String> list =  new ArrayList<String>();
Collections.addAll(list, ids);
```
Alternatively you can contruct presized ArrayList.

Reply:
Thank you,

I will be removing this portion since it is outdated. But I will consider this and your other comment for future uses of List and ArrayList :-)